ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -40,13 +42,21 @@ public override async Task<IActionResult> OnGetAsync(int? id)
             }
 
             NotificationId = Notification.NotificationId;
+            NotificationStatus = Notification.NotificationStatus;
+            Notification.SetFullValidation(NotificationStatus, isBeingSubmitted);
             Patient = Notification.PatientDetails;
 
             if (Patient == null)
             {
                 Patient = new PatientDetails();
             }
 
+            Patient.SetFullValidation(Notification.NotificationStatus, isBeingSubmitted);
+            if (Patient.ShouldValidateFull)
+            {
+                TryValidateModel(Patient, "Patient");
```

Comment:
I couldn't use GetType().Name since the type is PatientDetails and the errors are not displayed on front-end.
We either need to keep ViewModel and Model names consist, or use above workaroud

Reply:
I'd prefer if it was just called PatientDetails again, but I don't know why it was renamed in the first place!